app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,7 @@
+class AdminController < ApplicationController
+  def export_csv
```

Comment:
En fait on peux définir ça comme une `collection_action` dans ActiveAdmin… il doit y avoir moyen de le faire de façon globale pour toutes les pages (j’ai pas complètement cherché). Ça éviterait de créer un nouveau controller juste pour ça. 

(D’un autre côté, sortir des trucs de AA, ça ne me dérange pas plus que ça 😅 )

Reply:
Ah, mais d’ailleurs, si on arrive à ajouter une `collection_action` pour tous les modèles d’un coup, il faut aussi ajouter le `action_item` partout. Ça doit être possible, je regarde.

Reply:
Ah, en fait je l’avais déjà fait pour AdminArchivable :

```
module CsvExportable
  def self.included(dsl)
    dsl.action_item :export_csv, method: :post do
      path = polymorphic_path([:export_csv, :admin, resource_class.model_name.collection])
      link_to "Export CSV", path, method: :post
    end

    ## Using `.send` because the ResourceDSL methods are (wrongly) private
    # See https://github.com/activeadmin/activeadmin/issues/3673#issuecomment-291267819
    dsl.send(:collection_action, :export_csv,  method: :post) do
      CsvJob.perform_later(resource_class.model_name.name, current_user)
    end
  end
end
```

Ça marche presque tout seul, y’a plus qu’à `include CsvExportable` dans `admin/match.rb`